cmd/utils.go

```diff
@@ -991,56 +991,56 @@ func getLatestVersion() string {
 	resp, err := http.Get(LatestVersionURL)
 	if err != nil {
 		Warning.log("Unable to check the most recent version of Appsody in GitHub.... continuing.")
-		version = "none"
 	} else {
 		url := resp.Request.URL.String()
-		r, _ := regexp.Compile(`\d.\d.\d`)
+		r, _ := regexp.Compile(`[\d]+\.[\d]+\.[\d]+$`)
 
 		version = r.FindString(url)
 	}
 	return version
 }
 
-func doVersionCheck(data []byte, old string, new string, file string) {
+func doVersionCheck() {
 	var latest = getLatestVersion()
-	if latest != "none" && VERSION != "vlatest" && VERSION != latest {
-		Info.log("*\n*\n*\n\nA new CLI update is available.\nPlease go to " + LatestVersionURL + " and update from " + VERSION + " --> " + latest + ".\n\n*\n*\n*")
-	}
-	output := bytes.Replace(data, []byte(old), []byte(new), -1)
-	err := ioutil.WriteFile(file, output, 0666)
-	if err != nil {
-		Warning.log("Error writing to config file")
-	}
+	var currentTime = time.Now().Format("2006-01-02 15:04:05 -0700 MST")
+	configFile = getDefaultConfigFile()
+
+	if latest != "" && VERSION != "vlatest" && VERSION != latest {
+		switch os := runtime.GOOS; os {
```

Comment:
Thanks for writing this, I think it is really cool, however in talking to Michele, the longer instructions for linux and windows would be better to point to the website doc instead. Mainly because there are some additional notes in the docs and they may change in the future.
Also, there are two variations of linux (debian and rhel) with different instructions that you cannot detect here.

Net: keep the case for "darwin", but remove linux and windows to result in the default for them.

Thanks!

Reply:
Done